### ISSUE TITLE
[Bug]: ecoharmonogram.pl some streets are not working

### ISSUE THREAD

**Issue by ryrzy** [mampfes/hacs_waste_collection_schedule#2066]
### I Have A Problem With:

A specific source

### What's Your Problem

Hello, 
i have problem with some streets in city 'Zabrze', for example Zabrze/Zabrze/Łąkowa/1 Zabrze/Zabrze/Mickiewicza/1 - both shows 2 extra args when i use EcoHarmonogramPL.py script, and they are that same! :) i think one is for houses second for flats. 
But the best is for Leśna street... there are 2 extra args and they are empty! :D 
All streets are working on city official www - https://miastozabrze.pl/dla-mieszkancow/odpady-komunalne/wyszukiwarki-ecoharmonogram-harmonogram-odbioru-odpadow-jak-segregowac/

### Source (if relevant)

ecoharmonogram.pl

### Relevant Configuration

```YAML
waste_collection_schedule:
  sources:
    - name: ecoharmonogram_pl
      args:
        town: Zabrze
        district: Zabrze
        street: Leśna
        house_number: xyz
        additional_sides_matcher:
```

### Checklist Source Error

- [X] Use the example parameters for your source (often available in the documentation) (don't forget to restart Home Assistant after changing the configuration)
- [X] Checked that the website of your service provider is still working
- [X] Tested my attributes on the service provider website (if possible)
- [X] I have tested with the latest version of the integration (master) (for HACS in the 3 dot menu of the integration click on "Redownload" and choose master as version)

### Checklist Sensor Error

- [X] Checked in the Home Assistant Calendar tab if the event names match the types names (if types argument is used)

### Required

- [X] I have searched past (closed AND opened) issues to see if this bug has already been reported, and it hasn't been.
- [X] I understand that people give their precious time for free, and thus I've done my very best to make this problem as easy as possible to investigate.

**Comment by bispawel** [comment]
Hi,
I have a similar problem. 
Only the payment date and the date of the next mixed waste collection are displayed.

waste_collection_schedule:
  sources:
    - name: ecoharmonogram_pl
      args:
        town: "Nadolice Wielkie"
        street: "Zbożowa"
        house_number: "xx"
        district: "Czernica"
        additional_sides_matcher: ""

I tried other addresses from neighboring towns and it's the same everywhere.
I've been working on this for 3 days, trying different configurations and I have no idea what I could do on my side.

**Comment by pawelhulek** [comment]
Should be fixed here: https://github.com/mampfes/hacs_waste_collection_schedule/pull/2128

**Comment by ryrzy** [comment]
fix from @pawelhulek fixed my problem, thank you :)

**Comment by ryrzy** [comment]
fix from @pawelhulek fixed my problem, thank you :)

**Comment by pawelhulek** [comment]
@bispawel Can you check yours ?

**Comment by bispawel** [comment]
I checked and unfortunately no changes :( (everything is up to date)

**Comment by ryrzy** [comment]
> @pawelhulek I checked and unfortunately no changes :( (everything is up to date)

did you change files from @pawelhulek fork?
zmieniłeś pliki na te z forka Pawła? Musisz to zrobić ręcznie póki co

**Comment by bispawel** [comment]
@ryrzy you're right, I didn't read it. Now I replaced "ecoharmonogram_pl.py" and "EcoHarmonogramPL.py" but still no changes. I checked another address from a neighboring town (in Czernica district) and it was the same, only mixed garbage and payment deadline. I tested the address for the city of Tychy from this tutorial https://www.youtube.com/watch?v=KEhCRrnwFno and it works. (unfortunately it was the same before replacing the files)
